src/ngraph/frontend/onnx_import/op/leaky_relu.cpp

```diff
@@ -42,7 +42,8 @@ namespace ngraph
                         std::make_shared<default_opset::Constant>(data->get_element_type(),
                                                                   data->get_shape(),
                                                                   std::vector<double>{alpha});
-                    return {std::make_shared<default_opset::Maximum>(data * alpha_node, data)};
+                    return {std::make_shared<default_opset::Maximum>(
```

Comment:
please look at this pr: https://github.com/NervanaSystems/ngraph/pull/4359

Reply:
thanks, #4359 is fully review now and I'll merge it

Reply:
done